loadable_apps/loadable_sample/wifiapp/binary_update.c

```diff
@@ -484,24 +485,19 @@ static void binary_update_new_version_test(void)
 		return;
 	}
 
-	ret = binary_update_reload(APP_NAME);
+	ret = binary_update_reload(bin_name);
 	if (ret != OK) {
 		return;
 	}
 
-	sleep(2);
+	sleep(3);
```

Comment:
It means that after reload, should we wait some seconds?
If then, we should guide how to use.